### PR TITLE
New games

### PR DESCRIPTION
**Pull request by alexpalms**

### CODE REVIEW

File: diambra/arena/env_settings.py [hunk 1]
```diff
@@ -241,10 +242,15 @@ def _sanity_check(self):
             check_val_in_list("characters[{}]".format(idx), self.characters[idx], char_list)
         check_num_in_range("outfits", self.outfits, self.games_dict[self.game_id]["outfits"])
         check_val_in_list("super_art", self.super_art, [None, 1, 2, 3])
-        check_val_in_list("fighting_style", self.fighting_style, [None, 1, 2, 3])
+        if self.game_id == "kof98umh":
+            check_val_in_list("fighting_style", self.fighting_style, [None, 1, 2, 3])
```
Comment (discordianfish) [inline diambra/arena/env_settings.py:246]: btw I recently learned about https://docs.pydantic.dev which would could help improving these validations. Just something to check out whenever

**Review by discordianfish — APPROVED** [review]

**Comment by discordianfish** [comment]
But see test failures

File: diambra/arena/env_settings.py (hunk 1 above)
Comment (alexpalms) [reply diambra/arena/env_settings.py:246]: > btw I recently learned about https://docs.pydantic.dev which would could help improving these validations. Just something to check out whenever

Thanks a lot for the suggestion, will add this as a task!

**Comment by alexpalms** [comment]
> But see test failures

Yes to be re-evaluate after the release of the new engine

**Pull request merged**